template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Solution/Solution1.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30114.105
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.0.0
```

Comment:
@baronfel could you please confirm the version to use?

Reply:
The version number for 17.0 GA release was 17.0.31903.59, so we should use that.